### PR TITLE
Improve class pages

### PR DESCRIPTION
**Pull request by vinistock**
Implement our design on the class page

### CODE REVIEW

File: lib/rdoc/generator/template/snapper/class.rhtml [hunk 1]
```diff
@@ -1,163 +1,113 @@
...
-      <%- end -%>
+            <% end %>
+          </section>
+        <% end %>
+
+        <% klass.methods_by_type(section).select { |_type, v| v.any? }.each do |type, visibilities|
+          visibilities.each do |visibility, methods| %>
+            <% next if methods.empty? %>
+
+            <section>
```
Comment (adamhollett) [inline lib/rdoc/generator/template/snapper/class.rhtml:53]: I love the use of sections. Maybe we could give them a top margin so things are a bit less cramped, like this:

Screenshot 2023-07-28 at 3 57 51 PM

Something like:

``` css
section {
  margin-top: 50px;
}
```

Or even better:

``` css
section {
  margin-top: calc(var(--space) * 3);
}
```

File: lib/rdoc/generator/snapper.rb [hunk 2]
```diff
@@ -147,7 +147,16 @@ def create_sidebar_page_entries(grouped_files, rel_prefix)
       def copy_internal_static_files
         options = { :verbose => $DEBUG_RDOC, :noop => @dry_run }
 
-        ["fonts/Inter-Regular.ttf", "css/snapper.css", "js/snapper.js"].each do |item|
+        [
+          "fonts/Inter-Black.ttf",
```
[the remark this answers is not in the document]
Comment (adamhollett) [reply lib/rdoc/generator/snapper.rb:151]: Just realized that we can fix this by using a monospace font for method names too.

**Review by adamhollett — APPROVED** [review]
This looks great!

File: lib/rdoc/generator/snapper.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (vinistock) [reply lib/rdoc/generator/snapper.rb]: I just pushed a commit to use system fonts. Let me know what you think!

File: lib/rdoc/generator/template/snapper/class.rhtml (hunk 1 above)
Comment (vinistock) [reply lib/rdoc/generator/template/snapper/class.rhtml:53]: Great suggestion!

**Pull request merged**